### PR TITLE
WIP: Design events page

### PR DESCRIPTION
**Pull request by YaronMiro**
#87

### CODE REVIEW

**Comment by YaronMiro** [comment]
**Progress status**
* Map component not finished yet.

*Full page:

selection_201

* Selected item design:

selection_202

**Comment by amitaibu** [comment]
The `li` design is a bit confusing I think. We can discuss

File: src/elm/Event.elm [hunk 1]
```diff
@@ -285,33 +285,63 @@ leafletMarkers model =
...
-              , div [ hidden (isFetched model.status)] [ text "Loading..."]
-              ]
-
-          , div []
-              [ div [class "h2"] [ text "Event list"]
-              , (viewFilterString address model)
-              , (viewListEvents address model)
-              ]
-          ]
+  div
```
Comment (amitaibu) [inline src/elm/Event.elm]: Maybe we should split it to smaller functions?

**Comment by amitaibu** [comment]
Probably needs a re-roll :)

**Comment by YaronMiro** [comment]
@amitaibu 
I'm aware but it's not worth the time to resolve it.
That's why I'm currently working on a different branch and I'm porting the design from this PR to the new one.
this is the [relevant](https://github.com/Gizra/elm-hedley/pull/123) PR still work in progress. I will close this pr once I have all of the design on the new branch.

**Comment by YaronMiro** [comment]
Closing in favor of #123

**Pull request closed without merging**